lib/context.js

```diff
@@ -58,6 +61,28 @@ class Context {
   get isBot() {
     return this.payload.sender.type === 'Bot';
   }
+
+  /**
+   * Reads the plugin configuration from the given YAML file in the `.github`
+   * directory of the repository.
+   *
+   * @example
+   *
+    // Load config from .github/myplugin.yml in the repository
+    const config = await context.config('myplugin.yml');
+
+    if(config.close) {
+      context.github.issues.edit(context.issue({state: 'closed'}));
+    }
+   *
+   * @param {string} fileName Name of the YAML file in the `.github` directory
+   * @return {Promise<Object>} Configuration object read from the file
+   */
+  async config(fileName) {
+    const params = this.repo({path: path.join('.github', fileName)});
```

Comment:
While I don't believe that it is necessary, I feel that it is worth it to be opinionated ... to make the common case easy and the exceptional case possible but perhaps harder. What do you think about a system where it would default to `.github/` but could be overridden? Would you suggest adding an optional parameter for `basePath` or something?